### DISCUSSION TITLE
Error occurred when using filebeat to upload data to zincsearch.

### DISCUSSION THREAD

**Discussion by An-Bai** [zinclabs/zinc#938] [Q&A]
I have configured filebeat.yml correctly, but it is not working properly.
Test OK:
(base) lhqs@lhqs:~/app/filebeat$ ./filebeat test output
elasticsearch: http://192.168.1.100:4080/es/...
  parse url... OK
  connection...
    parse host... OK
    dns lookup... OK
    addresses: 192.168.1.100
    dial up... OK
  TLS... WARN secure connection disabled
  talk to server... OK
  version: 8.11.3
  
Error message:
  Failed to connect to backoff(elasticsearch(http://192.168.1.100:4080/es/)): Connection marked as failed because the onConnect callback failed: error loading template: failed to load template: couldn't load template: 400 Bad Request: {"error":"expected { character for map value"}. Response body: {"error":"expected { character for map value"}
2023-12-21T01:53:49.238Z	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(elasticsearch(http://192.168.1.100:4080/es/)) with 15 reconnect attempt(s)

**Comment by hengfeiyang** [comment]
Thank you for report this, i need time to test with this version of filebeat.
